modules/oe_theme_helper/src/Plugin/field_group/FieldGroupFormatter/InPageNavigationBase.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_helper\Plugin\field_group\FieldGroupFormatter;
+
+use Drupal\field_group\FieldGroupFormatterBase;
+
+/**
+ * Base class for In-page navigation field groups.
+ */
+abstract class InPageNavigationBase extends FieldGroupFormatterBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function settingsForm() {
+    $form = parent::settingsForm();
+    // Set label required, because we use it for links and navigation label.
+    $form['label']['#required'] = TRUE;
```

Comment:
If we are only going to use the label, then we should hide the Id and extra classes fields.